src/builtins/BuiltinTemporal.cpp

```diff
@@ -591,6 +591,48 @@ static Value builtinTemporalDurationPrototypeAbs(ExecutionState& state, Value th
     return TemporalDuration::createTemporalDuration(state, std::abs(duration->getYear()), std::abs(duration->getMonth()), std::abs(duration->getWeek()), std::abs(duration->getDay()), std::abs(duration->getHour()), std::abs(duration->getMinute()), std::abs(duration->getSecond()), std::abs(duration->getMillisecond()), std::abs(duration->getMicrosecond()), std::abs(duration->getNanosecond()), nullptr);
 }
 
+static Value builtinTemporalPlainMonthDayConstructor(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    if (!newTarget.hasValue()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, ErrorObject::Messages::New_Target_Is_Undefined);
+    }
+
+    int referenceISOYear = 1972;
+
+    if (argc >= 4) {
+        referenceISOYear = argv[3].asInt32();
```

Comment:
Is asInt32 checked somewhere? I mean if the user gives an incorerct value, does it throw an exception?